### PR TITLE
Created the User class

### PR DESCRIPTION
**Pull request by umeshtha**
HAHA! I DONE IT CALLUM. I FINALLY FINISHED DEVELOPMENT ON MY QUANTUM CARBURETOR!!

### CODE REVIEW

File: src/main/java/com/callumesh/bookingsystem/models/User.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+    //Attributes
+    private long mId;
+    private String mName;
+    private String mEmail;
+    private int mPhoneNo;
+    private String mUserType;
+
+    //Constructors
+    public User (long id, String name, String email, int phoneNo) {
```
Comment (VariableClass) [inline src/main/java/com/callumesh/bookingsystem/models/User.java]: Are users required to have both an e-mail and phone number to register on the system? The answer _can_ be yes :-)

File: src/main/java/com/callumesh/bookingsystem/models/User.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
```
Comment (VariableClass) [inline src/main/java/com/callumesh/bookingsystem/models/User.java]: Also, why is it possible to create a user without a type?

Comment (umeshtha) [reply src/main/java/com/callumesh/bookingsystem/models/User.java]: See when i was creating this i was kinda in a hurry so i had it in my head that user type is only needed if the user was staff but thinking about it properly i realize that "customers" can be the user type for users that arent staff.

File: src/main/java/com/callumesh/bookingsystem/models/User.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+ * @since       1.0
+ */
+
+public class User{
+
+    //Attributes
+    private long mId;
+    private String mName;
+    private String mEmail;
+    private int mPhoneNo;
```
Comment (VariableClass) [inline src/main/java/com/callumesh/bookingsystem/models/User.java]: Sorry; just seen the phone number is an `int`, should be a `String`. My bad; I'll approve as soon as you commit again. :-)

**Pull request merged**